tests/integration_test.sh

```diff
@@ -1,22 +1,26 @@
 #!/bin/bash
+set -ex 
 
 EXAMPLES=('https://github.com/near-examples/token-contract-as'
           'https://github.com/near-examples/counter'
           'https://github.com/near-examples/wallet-example'
           'https://github.com/near-examples/guest-book'
           'https://github.com/near-examples/chat'
-          'https://github.com/nearprotocol/chess'
+          'https://github.com/nearprotocol/near-chess'
           )
 
 # TODO=('https://github.com/nearprotocol/corgis'
 #       )
+
+cd examples || mkdir examples && cd examples;
+
 for i in "${EXAMPLES[@]}"
 do
-  mkdir example
```

Comment:
let's echo the project which is being tested so that we see what fails

Reply:
I add `set -ex` so that it prints out each command 😉